client/src/ContextCard.tsx

```diff
@@ -137,6 +141,7 @@ export default function CurrentContext() {
               </Tooltip>
             )}
           </Box>
+          <Box><b>Skip TLS Verify:</b> {ocOptions.skipTlsVerify}</Box>
```

Comment:
I modified the `ContextCard` to display the current `skipTlsVerify` state, is this alright?